libnmstate/nm/connection.py

```diff
@@ -202,6 +205,27 @@ def _active_connection_callback(self, src_object, result, user_data):
                 self._reset_profile()
                 time.sleep(0.1)
                 self.safe_activate_async()
+            elif (
+                # pylint: disable=no-member
+                isinstance(e, nmclient.GLib.GError)
+                and e.domain == NM_MANAGER_ERROR_DOMAIN
+                and (
+                    self._con_profile.is_type(
+                        nmclient.NM.SETTING_OVS_INTERFACE_SETTING_NAME
+                    )
+                    or self._con_profile.is_type(
+                        nmclient.NM.SETTING_OVS_PORT_SETTING_NAME
+                    )
+                    or self._con_profile.is_type(
+                        nmclient.NM.SETTING_OVS_BRIDGE_SETTING_NAME
+                    )
+                )
+                # pylint: enable=no-member
+            ):
+                self._mainloop.quit(
+                    "Missing 'NetworkManager-ovs' plugin"
+                    " to handle device={}".format(self.devname)
```

Comment:
You should be able to use `f" to handle device={self.devname}"` instead of using the `.format()` method.

Reply:
Done! Thanks